packages/compartment-mapper/src/zip/writer.js

```diff
@@ -26,6 +26,7 @@ export class ZipWriter {
    * }?} options
    */
   write(name, content, options = {}) {
+    // @ts-ignore
```

Comment:
What does TS dislike here? Perhaps capture in a comment above.

Reply:
I don't understand what TS doesn't like here.

Here's hoping I find time to look into it.

Reply:
Ah... the `{...}?` says that the `options` arg is optional; you can't destructure `null` / `undefined`:

```
> const { comment = "" } = null;
Uncaught TypeError: Cannot read property 'comment' of null
```

```
> const { comment = "" } = undefined;
Uncaught TypeError: Cannot read property 'comment' of undefined
```

But we have `options = {}` so options is never undefined. I should be able to fix the type...